sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -621,7 +622,7 @@ class SparkSession private(
   }
 
   private[sql] def table(tableIdent: TableIdentifier): DataFrame = {
```

Comment:
This is also called by the public `table` API, so this bug affect the public interface too.

Reply:
Yeah. many public APIs and DDL command processing are based on this internal API.